pheps/phep-9999.md

```diff
@@ -0,0 +1,145 @@
+```
+PHEP: 9999
+Title: PyHC Package Tiering
+Author: Julie Barnum <julie.barnum@lasp.colorado.edu> <https://orcid.org/0000-0001-8755-0694>
+Discussions-To: https://github.com/heliophysicsPy/standards/pull/31
+Revision: 1
+Status: Draft
+Type: Process
+Content-Type: text/markdown; charset=UTF-8; variant=CommonMark
+Created: 
+Post-History: 09-July-2024 
```

Comment:
This needs to be updated with the April push (and then with the date of whatever push adds the April push :)

Reply:
Ope, good catch... I kind of forgot this section existed!